CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
    * CHANGED: Avoid temporary std::string in some places in serializers [#5674](https://github.com/valhalla/valhalla/pull/5674)
    * ADDED: linux-aarch64 wheel for Python releases [#5670](https://github.com/valhalla/valhalla/pull/5670)
    * ADDED: PyPI `sdist` for python to install the bindings from source [#5649](https://github.com/valhalla/valhalla/pull/5649)
+   * CHANGED: STL-like casing for `GraphId` methods [#5678](https://github.com/valhalla/valhalla/pull/5678)
```

Comment:
nit: I would love to see something like "renamed `GraphId::Is_Valid()` -> `GraphId::is_valid()` and `GraphId::Tile_Base` -> `GraphId::tile_base`" here for folks like me who read changelog when something breaks after an update

Reply:
how about putting those details into the PR description?